papis/commands/export.py

```diff
@@ -114,13 +115,23 @@ def run(documents: List[papis.document.Document], to_format: str) -> str:
     help="Format for the document.",
     type=click.Choice(available_formats()),
     default="bibtex",)
+@papis.cli.bool_flag(
+    "-p",
+    "--append",
```

Comment:
The `-p` shorthand for `--append` is something I saw in `papis tag` and `papis update`, although in those it's not a bool flag.